server/views/pages/visit/summary.njk

```diff
@@ -243,24 +241,26 @@
 
           {% if visitHistoryDetails.eventsAudit %}
             <div class="moj-timeline">
+              {% set itemCount = 1 %}
```

Comment:
Keeping a separate count here because `loop.index` unsuitable as we don't show all items in `eventAudit`.